docs/en/cloud/AWS-EMR.md

```diff
@@ -31,93 +31,192 @@ Blob Store.
-Additional options can be seen in the comments at the top of the bootstrap script.
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+{% accordion setup create cluster %}
+  {% collapsible Run the aws create-cluster command with the bootstrap action %}
+The `create-cluster` command requires passing in multiple flags to successfully execute:
+- `release-label`: The version of EMR to install with.
+The current version of Alluxio is compatible with `emr-5.25.0`.
+- `custom-ami-id`: The Alluxio Marketplace Enterprise Edition AMI ID.
+To find the latest AMI ID for a particular region, run the following AWS CLI command:
+```console
+aws ec2 describe-images \
+--owners 'aws-marketplace' \
+--filters 'Name=product-code,Values=6axlevynmkcnoq7ximzar80bx' \
+--query 'sort_by(Images, &CreationDate)[-1].[ImageId]' \
+--output text \
+--region us-east-1
+```
+where the value for `--region` can be set to the desired region.
+
+- `instance-count`: The number of nodes to provision for the cluster.
+- `instance-type`: The instance type to provision with.
+Make sure you pick an instance type supported by the Alluxio marketplace AMI.
+Note that your account can be limited in the number of instances you can launch; check your instance limits [here](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Limits:).
+The default instance type for the AMI is `r4.4xlarge`.
+- `applications`: Specify `Name=Spark Name=Presto Name=Hive` to bootstrap the three additional services
+- `name`: The EMR cluster name
+- `bootstrap-actions`:
+  - `Path`: The path to the bootstrap script, hosted in a publicly readable S3 bucket: `s3://alluxio-public/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh`
+  - `Args`: The arguments passed to the bootstrap script.
+  The first argument, the root UFS URI, is required.
+  This S3 URI designates the root mount of the Alluxio file system and should be of the form `s3://bucket-name/mount-point`.
+  The mount point should be a folder; follow [these instructions](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-folder.html) to create a folder in S3.
+  You can also specify additional Alluxio properties as a delimited list of key-value pairs in the format `key=value`.
+  For example, `alluxio.user.file.writetype.default=CACHE_THROUGH` instructs ALluxio to write files synchronously to the underlying storage system.
+  See more about [write type options](https://docs.alluxio.io/ee/user/stable/en/Architecture-DataFlow.html?q=write%20types#data-flow-write).
+- `configurations`: The path to the configuration json file, also hosted in a publicly readable S3 bucket: `s3://alluxio-public/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.json`
+- `ec2-attributes`: EC2 settings to provide, most notably the name of the key pair to use to connect to the cluster
+
+Below is a sample command with all of the above flags populated:
 ```console
-$ aws emr create-cluster \
+aws emr create-cluster \
```

Comment:
if using `console` block type the command should start with `$`

Reply:
this seems counterintuitive to me since now the console blocks are not copyable, but if everything else is like this...
seems like we need a static check to enforce this

Reply:
It's difficult because `console` is supposed to represent commands and their output. take this `ls` example

```console
$ ls -l ~
total 52240
-rw-r--r--   1 zac-alluxio  staff  26731144 Oct 30 16:07 2.1.0-RC1-Report.html
drwx------@  4 zac-alluxio  staff       128 Apr  4  2019 Applications
drwx------+ 23 zac-alluxio  staff       736 Nov 11 16:27 Desktop
drwx------+ 22 zac-alluxio  staff       704 Nov 20 11:11 Documents
drwx------@ 36 zac-alluxio  staff      1152 Dec  4 11:11 Downloads
drwx------@ 76 zac-alluxio  staff      2432 Oct 10 13:30 Library
drwx------+  4 zac-alluxio  staff       128 Oct 10 13:08 Movies
drwx------+  5 zac-alluxio  staff       160 Oct 10 13:08 Music
drwx------+  5 zac-alluxio  staff       160 Oct 10 12:44 Pictures
drwxr-xr-x+  4 zac-alluxio  staff       128 Jan  9  2019 Public
drwxr-xr-x   4 zac-alluxio  staff       128 Sep 10 10:16 VirtualBox VMs
-rw-r--r--   1 root         staff       484 Oct 15 13:13 chectl-install.log
-rw-r--r--   1 zac-alluxio  staff      3147 Jan  9  2019 key.txt
drwxr-xr-x  30 zac-alluxio  staff       960 Dec  3 16:06 projects
drwxr-xr-x   5 zac-alluxio  staff       160 Sep  9 10:05 sandbox
drwxr-xr-x   3 zac-alluxio  staff        96 Jan 14  2019 vms
```
Notice the different highlighting for lines w/ and w/o the `$`